api/v1alpha4/machinedeployment_types.go

```diff
@@ -283,3 +287,11 @@ type MachineDeploymentList struct {
 func init() {
 	SchemeBuilder.Register(&MachineDeployment{}, &MachineDeploymentList{})
 }
+
+func (in *MachineDeployment) GetConditions() Conditions {
```

Comment:
This is causing linter errors.
You should use m instead of in (same receiver name as existing methods)

Reply:
That fixed. Thanks for pointing that out.